### PR TITLE
feat: add JWT expiration

### PR DESCRIPTION
**Pull request by Harasz**
### Summary

|             |   |
|-------------|---|
| Description | Add expiration to JWT |
| Jira issue  | https://energyweb.atlassian.net/browse/ICL-305  |

#### Type of request

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

### List of Features

- JWT expiration

### CODE REVIEW

File: packages/jwt/src/JWTEIP191.ts [hunk 1]
```diff
@@ -48,7 +48,7 @@ export class JWT extends JwtBase implements IJWT {
    */
   async sign(
     payload: string | JwtPayload,
-    { issuer, subject, noTimestamp }: JwtSignOptions = {}
+    { issuer, subject, noTimestamp, expirationTimestamp }: JwtSignOptions = {}
```
Comment (JGiter) [inline packages/jwt/src/JWTEIP191.ts:51]: There is also standard prop of JWT `expiresOn`. Could we use it?

**Review by jrhender — APPROVED** [review]
Looks good to me

File: packages/jwt/src/JWTEIP191.ts (hunk 1 above)
Comment (Harasz) [reply packages/jwt/src/JWTEIP191.ts:51]: Do you mean `exp`? I think `expirationTimestamp` is more clear

Comment (JGiter) [reply packages/jwt/src/JWTEIP191.ts:51]: Yes, the standard name is `exp`. What I meant is property of `jsonwebtoken` https://www.npmjs.com/package/jsonwebtoken#jwtsignpayload-secretorprivatekey-options-callback. It should also validate expiration

Comment (Harasz) [reply packages/jwt/src/JWTEIP191.ts:51]: I see `expiresIn` but it expect different value. `jsonwbetoken` want a seconds after which token is expired, but I expect a timestamp after which token is expired. Can be misleading for some peoples if we agree on the same name convention

**Pull request merged**